src/content/ja/basic-workflows/ksampler-advanced.md

```
@@ -0,0 +1,158 @@
+![](https://gyazo.com/814b1585c344ea2651ffe3bf16b95a0d){gyazo=image}
+
+図のように、前半 0 ~ 10stepは🟪Ksample (Advanced)、11~20stepは🟨KSampler (Advanced)で行ってみましょう。
+
+![](https://gyazo.com/d57cb22d3d85f90010815d19d45bb638){gyazo=image}
+
+[](/workflows/basic-workflows/ksampler-advanced/SD1.5_KSampler_(Advanced)-divide.json)
+
+2つに分割しても生成される画像が全く同じであるのがわかると思います。
+パラメータが少しややこしいので、丁寧に見ていきます。
+
+{% mediaRow img="https://gyazo.com/05f6a71f5fe47d5c2257bce31a015d2b{gyazo=image}", width=33, align="left" %}
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)